upcloud_api/cloud_manager/base.py

```diff
@@ -38,11 +38,13 @@ def request(self, method, endpoint, body=None):
         else:
             json_body_or_None = None
 
+        call_timeout = timeout if timeout != -1 else self.timeout
```

Comment:
When timeout is None, this could be stated as: call_timeout = timeout or self.timeout